src/Api/Repository.php

```diff
@@ -90,7 +90,8 @@ public function createApi($apiName, $host = null)
             $config['errors'],
             $config['providers'] ?? [],
             $config['controllers']['connection'] ?? null,
-            $config['controllers']['transactions'] ?? true
+            $config['controllers']['transactions'] ?? true,
+            $config['model-namespace'] ?? 'App'
```

Comment:
The default needs to be `null` - indicating the default application namespace. This might not be `App` because the namespace might have been changed. Laravel provides a way to work out what the application's namespace is, and that should be used if the model namespace is `null` - rather than defaulting to a hard-coded string of `App`.

Reply:
Worth saying that this also ensures it is non-breaking. Anyone upgrading will effectively have `null` as their setting which is correct as we'd then work out the default namespace. If this was left as `App` and someone upgraded an application that doesn't use `App`, their generators wouldn't work correctly.

Reply:
first of all, thank you for your detailed explanation, I really appreciate it.
I have amended the code and pushed a commit